DNN Platform/Library/Security/PortalSecurity.cs

```diff
@@ -668,6 +669,14 @@ public void SignIn(UserInfo user, bool createPersistentCookie)
                 var authCookie = HttpContext.Current.Response.Cookies[FormsAuthentication.FormsCookieName];
                 if (!string.IsNullOrEmpty(authCookie?.Value))
                 {
+                    authCookie.SameSite = SameSiteMode.Strict;
+                    if (settings.SSLEnabled)

```

Comment:
Should we be only setting this for SSLEnabled & SSLEnforced?

Reply:
Secure cookies are sent along only with requests that travel over a secure connection; usually this means a HTTPS connection.

Reply:
This should use `UrlUtils.IsSecureConnectionOrSslOffload` to check if the current request is secure